spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -37,6 +37,7 @@
 import java.util.Comparator;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
```

Comment:
There is no need to fix v3.4 as it's deprecated https://github.com/apache/iceberg/pull/14122